lib/pong_game/application.ex

```diff
@@ -14,7 +14,10 @@ defmodule PongGame.Application do
       PongGameWeb.Endpoint
       # Starts a worker by calling: PongGame.Worker.start_link(arg)
       # {PongGame.Worker, arg},
+      # {PongGameWeb.Game, []}
     ]
+    # start genserver ? ???
```

Comment:
Did you want to add a comment saying that saying this may not be the right place to call start link

Also if you wanted to look at it, i think there may be a proper way here https://hexdocs.pm/elixir/Supervisor.html#module-examples